examples/semantic_segmentation.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>Maybe add this? (nit, up to you to change if you agree)</p><p> </p><p>"... version of those libraries or run the <code>pip install</code> command above with the <code>--upgrade</code> flag"</p><p>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>